src/components/ItemCard.css

```diff
@@ -0,0 +1,15 @@
+.Item-card {
```

Comment:
is Item-card a component? or why is it capital

Reply:
Indeed, It´s a component.